estela-web/src/components/ProjectSettingsPage/index.tsx

```diff
@@ -0,0 +1,407 @@
+                                                    onChange={this.handleNameChange}
+                                                    className="h-14 rounded-lg border-[#4D47C3]"
+                                                />
+                                            </Form.Item>
+                                            <Form.Item label="Category" name="category">
+                                                <Select
+                                                    className="h-14 rounded-lg border border-[#4D47C3]"
+                                                    placeholder={<div className="">Select...</div>}
+                                                >
+                                                    <>
+                                                        <Option
+                                                            className="text-gray-500 hover:bg-[#F6FAFD] selection:border-[#4D47C3] selection:text-[#4D47C3]"
+                                                            value="notspecified"
+                                                        >
+                                                            Not Specified
+                                                        </Option>
+                                                        <Option
+                                                            className="text-gray-500 hover:bg-[#F6FAFD] selection:border-[#4D47C3] selection:text-[#4D47C3]"
+                                                            value="ecommerce"
+                                                        >
+                                                            E-commerce
+                                                        </Option>
+                                                        <Option
+                                                            className="text-gray-500 hover:bg-[#F6FAFD] selection:border-[#4D47C3] selection:text-[#4D47C3]"
+                                                            value="logistics"
+                                                        >
+                                                            Logistics
+                                                        </Option>
+                                                        <Option
+                                                            className="text-gray-500 hover:bg-[#F6FAFD] selection:border-[#4D47C3] selection:text-[#4D47C3]"
+                                                            value="finance"
+                                                        >
+                                                            Finance
+                                                        </Option>
+                                                        <Option
+                                                            className="text-gray-500 hover:bg-[#F6FAFD] selection:border-[#4D47C3] selection:text-[#4D47C3]"
```

Comment:
Use predefined colors set in tailwind.conf.js
Use `text-estela` instead of text-[#4D47C3]